dev/gulpfile.js

```diff
@@ -32,7 +32,8 @@ let arcsBuild = async (path) => {
     const options = {cwd: resolve(path), stdio: 'inherit'};
     await execSync('npm install', options);
 
-    await execSync(`node tools${sep}sigh.js`, options);
+    await execSync(`node tools${sep}sigh.js webpack`, options);
```

Comment:
FYI, instead of inserting `${sep}` there is a `path.normalize(...)` that can fix paths for the current platform.